### PR TITLE
Adding the list of names that sit behind the CoC email address

### PR DESCRIPTION
**Pull request by stephenrwalli**
Ensure folks know who sits on the code-of-conduct reflector in case the concern being reported is with someone on the governing board itself.

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -41,6 +41,20 @@ This Code of Conduct applies within all community spaces, and also applies when
...
+Facebook | Michael Cheng | mfc@fb.com
+Google | Nelly Porter | nellyporter@google.com
+Huawei | Peixin Hou | peixin.hou@huawei.com
+Intel | Ron Perez | ronald.perez@intel.com
+Microsoft | Stephen Walli | stephen.walli@microsoft.com
+Oracle | Sergio Leunissen | sergio.leunissen@oracle.com
+Red Hat | Mike Bursell | mbursell@redhat.com
+TAC Chair | Dave Thaler | dthaler@microsoft.com
+General Member Rep | Richard Searle  | richard.searle@fortanix.com
+
```
Comment (mwklein) [inline code-of-conduct.md:57]: Ask the PR to be updated to include Accenture as well.

Comment (cetola) [reply code-of-conduct.md:57]: @mwklein can do, shall I add your email?

Comment (mwklein) [reply code-of-conduct.md:57]: Yes, my email can be included.

**Comment by cetola** [comment]
I'll add Michael after this in another commit.

**Pull request merged**